index.bs

```diff
@@ -168,15 +168,16 @@ The <dfn method for=FileSystemHandle>requestPermission(|descriptor|)</dfn> metho
 ## The {{FileSystemFileHandle}} interface ## {#api-filesystemfilehandle}
 
 <xmp class=idl>
-dictionary FileSystemCreateWriterOptions {
+dictionary FileSystemCreateWritableOptions {
+  unsigned long long position = 0;
   boolean inPlace = false;
   boolean keepExistingData = false;
```

Comment:
I wonder if "keepExistingData" should be more of an enum like you suggested so we can have append/truncate/overwrite modes, where append would start at the end of the file, and truncate/overwrite would start at the end of the file... That combined with possibly a seek() method (and no other explicit initial position), or the ability to pass in a position to getWriter almost feels like it might be the best of both worlds. I suppose the only thing missing if we don't have an explicit initial position or seek method is the ability to pipe into the middle of an existing file, but that perhaps is an okay compromise?

Reply:
We do now have a `seek` method without an explicit initial position. The enum for creating the stream is a good idea.

Tracking in #141